### ISSUE TITLE
[swf] html5 Canvas : mask made with a simple shape is not working

### ISSUE THREAD

**Issue by loudoweb** [openfl/openfl#1831]
I have some MovieClips in a swf made with Animate.

First I get a MovieClip from my swf library:
`mc = Assets.getMovieClip(bundleName + ":" + template);`

Now, when I want to get one of his child, I do:
`display = mc.getChildByName('nameMc');`

Then, if I set visible to false, like that:
`display.visible = false;`

The display object is still visible but it should be not visible. It seems to happen for some kind of objects, not all of them and on Canvas mode (seems to work in WebGL).

**Comment by mastef** [comment]
Might be related to #1769 

![click1](https://user-images.githubusercontent.com/762297/36653956-c5e1c840-1aeb-11e8-8cd6-ccdb5ceef4b7.png)
![click2](https://user-images.githubusercontent.com/762297/36653957-c61ad608-1aeb-11e8-88d9-271ab5161de4.png)

When clicking on the left element the renderMask commands for `LINE_STYLE`, `BEGIN_FILL` and `END_FILL` are not implemented.

**Comment by mastef** [comment]
In CanvasGraphics.hx try adding this to the `renderMask()` switch statement : 

```
                    case END_FILL:
                        data.readEndFill ();
                        positionX = 0;
                        positionY = 0;

                    case LINE_STYLE:
                        data.readLineStyle ();

                    case BEGIN_BITMAP_FILL, BEGIN_FILL, BEGIN_GRADIENT_FILL:
                        context.beginPath ();
                        if (type == BEGIN_BITMAP_FILL) data.readBeginBitmapFill ();
                        if (type == BEGIN_GRADIENT_FILL) data.readBeginGradientFill ();
                        if (type == BEGIN_FILL) data.readBeginFill ();

                    default:
                        lime.utils.Log.warn (type + " on masks is not implemented");
                        data.skip (type);
```

@guruas3 see if that also helps in your case?

**Comment by guruas3** [comment]
@loudoweb Please check again as the issue seems to be fixed in latest 8.0.1 release.
